package.json

```diff
@@ -12,12 +12,36 @@
     "url": "https://github.com/stoplightio/http-spec"
   },
   "license": "Apache-2.0",
-  "main": "src/index.ts",
   "files": [
     "**/*"
   ],
   "engines": {
-    "node": ">=10"
+    "node": "^12.20 || >=14.13"
+  },
+  "type": "commonjs",
+  "main": "./index.js",
+  "types": "./index.d.ts",
+  "exports": {
```

Comment:
alternatively, if we reallllly want to avoid breaking changes, we'd need to define each entry based on our usage in platform, prism, and everywhere else, but I believe these defaults are good. 
We should be really importing anything very deeply nested